### PR TITLE
[pprint]: Add docstring about `PrettyPrinter.underscore_numbers` parameter

### PR DESCRIPTION
**Pull request by KrySeyt**
pprint.PrettyPrinter.__init__ param underscore_numbers description added in method docstring.

Should I create NEWS entry?

### CODE REVIEW

File: Lib/pprint.py [hunk 1]
```diff
@@ -128,6 +128,9 @@ def __init__(self, indent=1, width=80, depth=None, stream=None, *,
         sort_dicts
             If true, dict keys are sorted.
 
+        underscore_numbers
+            If true, The number classes will be separated by underscores.
```
Comment (Eclips4) [inline Lib/pprint.py]:
```suggestion
            If true, the number classes will be separated by underscores.
```

**Review by Eclips4 — APPROVED** [review]
LGTM. Perhaps we can improve the wording, but I'm not a native English speaker 😢.
 cc @alexwaygood

**Review by AlexWaygood — APPROVED** [review]
Nice, thank you!

**Comment by Eclips4** [comment]
Thanks @KrySeyt for the PR!

**Pull request merged**